### PR TITLE
fix(diff): fix diff foreground color

### LINKED ISSUE #3: Text unreadable/unusable with diffing plugins
Pretty much unusable with `neogit` & `diffview`

Screenshot 2023-01-23 at 3 32 52 AM
Screenshot 2023-01-23 at 3 36 54 AM

### PR DESCRIPTION
**Pull request by kuntau**
### Before
Screenshot 2023-01-23 at 3 32 52 AM

### After
Screenshot 2023-01-23 at 4 54 32 AM

Fix #3

### CODE REVIEW

**Comment by AlexvZyl** [comment]
Thanks for the PR!  Though, I now regret making the `diff` colors the bright versions of the palette.  I do not think it looks good.

(Red and green is bright and yellow is base, since yellow generally seems brighter in this palette)

**Comment by kuntau** [comment]
Or just reverse to use without background color whichever you prefer. This way a bit harder to see `change` and `text` change

Screenshot 2023-01-23 at 5 12 33 AM

**Comment by kuntau** [comment]
> Thanks for the PR! Though, I now regret making the `diff` colors the bright versions of the palette. I do not think it looks good.
> 
> (Red and green is bright and yellow is base, since yellow generally seems brighter in this palette)

Yes, I have tried using the dim version here, and white~ `delete` fg
Screenshot 2023-01-23 at 5 17 18 AM

**Comment by AlexvZyl** [comment]
Dim looks better, but I think specific colors should be added for the diff.

File: lua/nordic/colors.lua [hunk 1]
```diff
@@ -109,9 +109,9 @@ palette.border_nb = palette.orange.base
 
 -- Diffs.
 palette.diff = {}
-palette.diff.add = palette.green.bright
-palette.diff.change = palette.yellow.base
-palette.diff.delete = palette.red.bright
+palette.diff.add = palette.green.dim
+palette.diff.change = palette.yellow.dim
+palette.diff.delete = palette.red.dim
```
Comment (AlexvZyl) [inline lua/nordic/colors.lua:114]: This is better for now.  Though, the `git` colors now needs to be changed to the bright versions again (and the yellow should be base).

**Comment by AlexvZyl** [comment]
This is fine for now, at least it is readable.  I will add colors specifically designed `diff` in the morning.

**Comment by kuntau** [comment]
Thank you!

**Pull request merged**